scss/utilities/mixins/_hero-banner-node.scss

```diff
@@ -12,8 +12,9 @@
 
 @mixin hero-banner-node {
 
-  article .content > div:first-child .hero-banner,
-  article #panels-ipe-content > section:first-child .hero-banner {
+  // Due to the way the rendering works on the twig templates there will always
+  // be hero-banner-0 as the first one.
+  #hero_banner-0 {
```

Comment:
@pookmish, this will likely have to change to match the new unique ID implementation you provided in the components. It should be `#hero-banner` instead of `#hero-banner-0` .